bridge/github/export.go

```diff
@@ -100,6 +100,17 @@ func (ge *githubExporter) ExportAll(repo *cache.RepoCache, since time.Time) (<-c
 		return nil, err
 	}
 
+	client, err := ge.getIdentityClient(user.Id())
+	if err != nil {
+		return nil, err
+	}
+
+	// query all labels
+	err = ge.cacheGithubLabels(client)
```

Comment:
you will need to have pass the context here as well after #190

Reply:
Exactly why i suggested a rebase before :)